x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -595,6 +595,28 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.diagnostic.enabled',
+    first_supported_version: '7.12',
```

Comment:
Should this be `7.12.1`?

Reply:
no because the Endpoint already supports it.  So if they have a 7.12.1 Kibana, they can still use the field with a 7.12 Endpoint.